### DISCUSSION TITLE
Filecoin at Chainlink Spring Hackathon

### DISCUSSION THREAD

**Discussion by kcnnn** [filecoin-project/community#511] [General]
Howdy I work for Truflation which is also taking part in the hackathon.  I think it would be cool to involve filecoin and the truflation oracle in a single project.  If anyone in the community or at filecoin is interested, let me know.

**Comment by lksbrssr** [comment]
Hi @kcnnn - please also consider posting this in the Chainlink IPFS Filecoin Discord channel :) Lots of builders, there, too
